contrib/release/start-release.sh

```diff
@@ -113,12 +113,12 @@ main() {
     else
       $DIR/prep-changelog.sh "$old_version" "$version"
     fi
-    git commit -a -s -m "Prepare for release $version"
```

Comment:
I'll probably just fix this issue by making it commit the authors separately during the next RC prep. Do we need to revert this?